source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -117,6 +117,105 @@ For a more data-efficient browser preload, we also recommend that you define the
 />
 ```
 
+## Image Sizes
+
+First be sure you have a good understanding of how the browser selects what image to load depending on the image rendered width and the image widths available to it in the srcset attribute ([this](https://ericportis.com/posts/2014/srcset-sizes/#part-2) link is a good start).
+
+We have added sensible defaults to image sizes on key components in the Home, Category, Search, Product pages. Figuring the correct `sizes` to set can be a daunting task, you need to know the different sizes available for your image and take into account the size it will be rendered as in relation to media breakpoints on the page and parent containers' `max-width`, `padding`and`margin` properties. To simplify this process we devised a smart way to determine these numbers in a very straightforward way.
```

Comment:
This block was dense. Here is a proposal to make it a bit easier to read. Also I think that having a title for the "method" would be good to directly link to it.

```suggestion
We have added sensible defaults to image sizes on key components in the Home, Category, Search, Product pages. Figuring the correct `sizes` to set can be a daunting task. You need to know the different sizes available for your image. You must also take into account the size it will be rendered as, in relation to media breakpoints on the page and parent containers' `max-width`, `padding` and `margin` properties.

### A method to determine image sizes

To simplify this process we devised a method to help you determine these numbers for your project.
```